core/src/test/java/org/apache/iceberg/jdbc/TestJdbcCatalog.java

```diff
@@ -102,12 +102,12 @@ protected JdbcCatalog catalog() {
   }
 
   @Override
-  protected boolean supportsNamespaceProperties() {
```

Comment:
Intellij flags this because `supportsNamespaceProperties()` is `true` by default in `CatalogTests`, so no need to override this method here